doc/api/errors.md

```diff
@@ -1019,6 +1019,12 @@ made to mark a stream and dependent of itself.
 
 A given index was out of the accepted range (e.g. negative offsets).
 
+<a id="ERR_INSPECTOR_ALREADY_ATTACHED"></a>
+### ERR_INSPECTOR_ALREADY_ATTACHED
+
+While using the `inspector` module, an attempt was made to connect a session
```

Comment:
Should this reuse the `ERR_INSPECTOR_ALREADY_CONNECTED` instead of creating a new error? From what I can tell, the only distinction is that if you connect from the same session twice, you get an `ERR_INSPECTOR_ALREADY_CONNECTED`, whereas if you connect from another session, you get an `ERR_INSPECTOR_ALREADY_ATTACHED`, but the cause are the same: there is a current connected session, whether it's the same one or a different one does not seem to make a big difference.

Reply:
Good point. I'm going to change this.